### PR TITLE
Enabe the use of multiple and statements. Fixes #78 Also Fixes #69

### LINKED ISSUE #69: ACCompiler does things that ACParser should do instead
`parse: sourceStream class: cls noPattern: pat context: ctx notifying: req ifFail: block`
Is implemented in both ACCompiler and ACParser. The implementations slightly differ but they do exactly the same thing. The compiler should probably not have this message at all but is somewhere listed as a Parser.

So Todo:
1. Remove the method from the compiler
2. Fix tests by adding ACParser (or the correct subclass) as `parserClass` instead of ACCompiler
3. See if the project works
4. Test the ACParser implementation of this method.
5. Refactor implementation.

If 3. fails then please leave a comment here on why it didn't work out as I thought it would.

### LINKED ISSUE #78: Multiple ANDs from Scenarios
### Story

AS A developer
I WANT TO be able to have multiple AND statements
IN ORDER TO specify steps more specifically

### PR DESCRIPTION
**Pull request by Askir**
I can maybe explain in person why this works and somebody helps me flesh out the tests this afternoon? I'm kind of surprised myself that it does work to be honest.

### CODE REVIEW

File: packages/acceptit-Core.package/ACScenarioSemantic.class/instance/scenario.withGiven.andLinebreak.andAndLinebreak.andWhen.andLinebreak.andAndLinebreak.andThen.andLinebreakAnd..st [hunk 1]
```diff
@@ -7,18 +7,23 @@ andAndLinebreak: and2
...
+	|return givens whens thens andResult1 andResult2 andResult3|
 	return := OrderedCollection new.
-	return add: (self value: aGivenRule).
+	givens := OrderedCollection new.
+	givens add: (self value: aGivenRule).
 	self value: linebreak1.
 	andResult1 := self value: and1.
-	(andResult1 isNil) ifFalse: [return add: andResult1].
-	return add: (self value: aWhenRule).
+	andResult1 do: [:each| givens add: each].
```
Comment (MarcelSchmidberger) [inline packages/acceptit-Core.package/ACScenarioSemantic.class/instance/scenario.withGiven.andLinebreak.andAndLinebreak.andWhen.andLinebreak.andAndLinebreak.andThen.andLinebreakAnd..st]: not really sure why it is still necessary to differentiate between statement and the respective and statements. Could they be entered into the method all as givens? At the end they are added all to the local var givens with no differentiation anyway

Comment (Askir) [reply packages/acceptit-Core.package/ACScenarioSemantic.class/instance/scenario.withGiven.andLinebreak.andAndLinebreak.andWhen.andLinebreak.andAndLinebreak.andThen.andLinebreakAnd..st]: See new method for explanation. its cleaner now

File: packages/acceptit-Core.package/ACTranslator.class/class/buildScenarioFrom..st [hunk 2]
```diff
@@ -4,7 +4,8 @@ buildScenarioFrom: aDictionary
 	|code|
 	
 	code := ((aDictionary at: #name) copyReplaceAll: (String space) with: ''), String cr,  String tab, '<scenario>'.
-	(aDictionary at: #codelines ) do: [ :line| 
-		code := code, String cr, String tab, 'self ', line first asString, ': ''', ((line second) copyReplaceAll: '''' with: ''''''), '''.' 
-		].
+	(aDictionary at: #codelines ) do: [ :verbStatements|
+		verbStatements do:[:line|
+			code := code, String cr, String tab, 'self ', line first asString, ': ''', ((line second) copyReplaceAll: '''' with: ''''''), '''.'
```
Comment (MarcelSchmidberger) [inline packages/acceptit-Core.package/ACTranslator.class/class/buildScenarioFrom..st]: extract in method

**Review by MarcelSchmidberger — APPROVED** [review]
some unclassified methods in the tests but thats something for later

**Pull request merged**